deploy-idsvr-aws-eks.sh

```diff
@@ -6,35 +6,35 @@ display_help() {
     echo "** DESCRIPTION **"
     echo -e "This script can be used to deploy Curity Identity Server in AWS Elastic kubernetes cluster. \n"
     echo -e "OPTIONS \n"
-    echo " --help      shows this help message and exit                                                                 "
-    echo " --install   creates a eks cluster & deploys curity identity server along with other components               "
-    echo " --start     starts up the environment                                                                        "
-    echo " --stop      shuts down the environment                                                                       "
-    echo " --delete    deletes the eks k8s cluster & identity server deployment                                         "
+    echo " --help                         shows this help message and exit                                                                 "
+    echo " --install                      creates a eks cluster & deploys curity identity server along with other components               "
+    echo " --start                        starts up the environment                                                                        "
+    echo " --stop                         shuts down the environment                                                                       "
+    echo " --load-balancer-public-ip      prints the public IP address of the load balancer                                                "
+    echo " --delete                       deletes the eks k8s cluster & identity server deployment                                         "
```

Comment:
... cluster and curity identity server deployment

Reply:
Fixed